strapi/strapi
# chore: set up preview feature boilerplate

### What does it do?

- Sets up boilerplate so we can start working on preview
- Only loads feature when a `preview` feature flag is passed

I copied the system we used for history that worked pretty well: we have one folder for the feature in the CM admin, and one in the CM server. This way it's pretty isolated, which would make it easy to turn it into a plugin in the future. Each folder contains the EE license because they're outside of the EE directory.

### Why is it needed?

- so we can start working on preview
- so we don't need to maintain a feature branch
### How to test it?

- Start Strapi. You should see no Preview-related logs, whether it's in the browser console or terminal.
- Set a `STRAPI_FUTURE_PREVIEW` env var to true in your getstarted app
- Start Strapi. You should see "Bootstrapping preview" logs in both the browser console and the terminal
